common/src/test/java/com/tc/runtime/MemoryPoolsTest.java

```diff
@@ -33,6 +33,8 @@ public MemoryPoolsTest() {
   }
 
   public void testMemoryPools() throws Exception {
+    // This test fails with java11 because of collection count assertion failure .
+    // It depends on the GC algortihm whether it will trigger GC or not on calling System.gc().
```

Comment:
The `assertTrue(collectorCount2 > collectorCount1);` at line 61 needs to be commented out and the comment at this line moved there to explain the assertion's removal.  The assertion is what's bogus about this test -- actual GC is not guaranteed when `System.gc()` is called and the assertion is based on the fallacy that GC will run when `System.gc()` is called.